### PR TITLE
use openapi explorer for /api/docs

### PR DESCRIPTION
**Pull request by machisuji**

### CODE REVIEW

File: app/views/api_docs/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<html>
+  <head>
+    <%= javascript_include_tag "openapi-explorer.min.js" %>
+  </head>
+  <body>
+    <openapi-explorer spec-url="/api/v3/spec.json"> </openapi-explorer>
```
Comment (machisuji) [inline app/views/api_docs/index.html.erb]: There's probably some path helper for `/api/v3/spec.json`. We'll leave that as an exercise for the reader.

**Comment by ulferts** [comment]
Just because I stumbled over this PR:

Please remember to remove all the Swagger leftovers throughout the code and the documentation as well as to adapt the documentation.

File: app/views/api_docs/index.html.erb (hunk 1 above)
Comment (Kharonus) [reply app/views/api_docs/index.html.erb]: resolved

**Comment by Kharonus** [comment]
You wanna have a final look, @machisuji?

**Review by Kharonus — APPROVED** [review]

**Comment by machisuji** [comment]
@Kharonus Good stuff! 👍

**Pull request merged**